lib/rmt/cli/products.rb

```diff
@@ -35,21 +38,87 @@ def list
   end
   map ls: :list
 
-  desc 'enable', 'Enable mirroring of product repositories by product ID or product string.'
+  desc 'enable TARGETS', 'Enable mirroring of product repositories by a list of product IDs or product strings.'
   option :all_modules, type: :boolean, desc: 'Enables all free modules for a product'
-  def enable(target)
-    change_product(target, true, options[:all_modules])
+  long_desc <<-REPOS
+
+Examples:
+
+`rmt-cli products enable SLES/15`
+
+`rmt-cli products enable 1575`
+
+`rmt-cli products enable SLES/15/x86_64,1743`
+
+`rmt-cli products enable --all-modules SLES/15`
+REPOS
+  def enable(*targets)
+    change_products(targets, true, options[:all_modules])
   end
 
-  desc 'disable', 'Disable mirroring of product repositories by product ID or product string.'
-  def disable(target)
-    change_product(target, false, false)
+  desc 'disable TARGETS', 'Disable mirroring of product repositories by a list of product IDs or product strings.'
+  def disable(*targets)
+    change_products(targets, false, false)
   end
 
   protected
 
+  def change_products(targets, set_enabled, all_modules)
+    targets = clean_target_input(targets)
+    raise RMT::CLI::Error.new('No product ids supplied') if targets.empty?
+
+    failed_targets = []
+    targets.each do |target|
+      change_product(target, set_enabled, all_modules)
+    rescue ProductNotFoundException => e
+      puts e.message
+      failed_targets << target
+    end
+
+    unless failed_targets.empty?
+      message = if set_enabled
+                  "Product(s) #{failed_targets.join(',')} could not be found and were not enabled."
```

Comment:
A little nitpick on the output: 
```suggestion
                  "Product(s) #{failed_targets.join(', ')} could not be found and were not enabled."
```

So there's a space between comma and target.